// Copyright 2018 The Chromium Authors
// Use of this source code is governed by a BSD-style license that can be
// found in the LICENSE file.
//
// See platform2/media_perception/mojom/media_perception.mojom for relevant
// comments on data fields.

syntax = "proto2";

package mri;

message SuccessStatus {
  optional bool success = 1;
  optional string failure_reason = 2;
}
